src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -138,6 +145,124 @@ public static DatasetModel populateDatasetModelFromDataset(
           metadataDataAccessUtils.accessInfoFromDataset(dataset, userRequest));
     }
 
+    if (include.contains(DatasetRequestAccessIncludeModel.SNAPSHOT_BUILDER_CONFIG)) {
+      datasetModel.snapshotBuilderConfig(
+          new SnapshotBuilderConfig()
```

Comment:
It didn't occur to me until now, but maybe `Config` isn't a good name for this model. The problem I see is that Config is used in the Java code to describe application configuration objects, so it may be confusing. Would `Settings` work for this concept?

Reply:
I like this name change, can we make sure it's consistently applied throughout to avoid confusion (such as extending to enum naming)?